ONSdigital/rasrm-acceptance-tests
# Setup vacancies

# Motivation and Context
Needed to add vacancies surveys to the data setup.  This is partially so we can write tests against it in the future if required, but mainly so we can easily test the vacancies surveys + secure message functionality manually in a dev environment.

Also added the setup of the vacancies survey in the survey service here.  All other surveys are setup via an SQL script in the survey service.  It doesn't seem practical to figure out the sql to create a new survey every time we onboard a survey, so I'm starting what I feel is the correct trend here.

# What has changed
- The small bit of rm-tools we use for data setup has been moved into this repo.  This is because the setup data should live with the thing that cares about it.
- Added vacancies survey, collection exercise and collection instrument setup to data setup
- Updated database script to set up vacancies in survey service.

# How to test?
- Run `make acceptance_tests` it should build as before.
- Follow the test steps in https://github.com/ONSdigital/response-operations-ui/pull/402 to make sure things appear as they should do.

# Links
https://trello.com/c/botlRPsE
